idl/partition_checksum.idl.hh

```diff
@@ -110,13 +110,20 @@ enum class node_ops_cmd : uint32_t {
      removenode_sync_data,
      removenode_abort,
      removenode_done,
+     replace_prepare,
+     replace_prepare_mark_alive,
+     replace_prepare_pending_ranges,
+     replace_heartbeat,
+     replace_abort,
+     replace_done,
 };
 
 struct node_ops_cmd_request {
     node_ops_cmd cmd;
     utils::UUID ops_uuid;
     std::list<gms::inet_address> ignore_nodes;
     std::list<gms::inet_address> leaving_nodes;
+    std::unordered_map<gms::inet_address, gms::inet_address> replace_nodes;
```

Comment:
I am guessing that the key is the node being replaced and the value is the node replacing, but I should be reading a comment, not guessing.